docs/sphinx/developers/tiling.txt

```diff
@@ -0,0 +1,129 @@
+Tiled reading and writing in Bio-Formats
+========================================
+
+This guide pertains to version 5.3 and later.
+
+Reading tiled images
+--------------------
+
+The reading of tiled images is straightforward and can be done in much the same way as reading a full image. In this case, to read an individual tile, we pass to the reader parameters for the x and y co-ordinates of the tile to read and the height and width of tile desired.
+
+::
+
+    byte[] tile = reader.openBytes(image, tileX, tileY, tileSizeX, tileSizeY);
+
+For Tiff based readers, if the image has specifically been written using tiles, then the tile width and height used can be found as below. These values can then be used with the above command to read the correct tiles individually.
+
+::
+
+    IFD tileIFd = reader.getIFDs().get(0);
+    int tileHeight = tileIFd.getIFDIntValue(IFD.TILE_LENGTH);
+    int tileWidth = tileIFd.getIFDIntValue(IFD.TILE_WIDTH);
+
+Introduction to tiled writing
+-----------------------------
+
+Tiled writing is currently supported for Tiff based formats.
+To setup an image writer to use tiling the following 2 API functions are provided:
+
+::
+
+	public int setTileSizeX(int tileSize) throws FormatException
+	public int setTileSizeY(int tileSize) throws FormatException
+
+Each function takes in an integer parameter for the desired tile size. As not all tile sizes are supported, the image writer will round the requested value to the nearest supported tile size. The return value will contain the actual tiling size which will be used by the writer.
+
+To find out the tiling size currently being used at any point there are 2 further API functions to get the current tile size for a writer. If tiling is not being used or is not supported then the full image height and width will be returned.
+
+::
+
+	public int getTileSizeX() throws FormatException
+	public int getTileSizeY() throws FormatException 
+
+The tiling parameters for writers must be set after the image metadata is set. An example of initializing a writer for tiling is show below
```

Comment:
Sometimes before code you end with a period, sometimes not; I can't quite spot the pattern.